test/range-counter-test.js

```diff
@@ -0,0 +1,31 @@
+const rangeCounter = require("../range-counter");
+const { expect } = require("chai");
+
+describe("range counter", () => {
+  it("should exist", () => {
+    expect(rangeCounter.getRangeCounter).to.exist;
+  });
+  it("should return range pairs", () => {
+    let mockData = [4, 5];
+    expect(rangeCounter.getRangeCounter(mockData)).to.include({'4-5':2});
+    mockData = [3, 3, 5, 4, 10, 11, 12];
```

Comment:
test data set need to have varying values. you may add test cases with single value, 2 values list with same element repeated.